roles/cluster_logging/templates/job-setup-configmap.yml.j2

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: job-setup-configmap
```

Comment:
Needs some labels as well, and a more descriptive name.

Reply:
Point being: these objects are 'global' in the `kube-ops` namespace, so a name like `kibana-index-pattern-setup` could be better.